src/custom/pages/AppBody/index.tsx

```diff
@@ -13,10 +12,5 @@ export const BodyWrapper = styled(BodyWrapperMod)`
 
 // The styled container element that wraps the content of most pages and the tabs.
 export default function AppBody({ children, className }: { children: React.ReactNode; className?: string }) {
-  return (
-    <BodyWrapper className={className}>
-      {children}
-      <Version />
-    </BodyWrapper>
-  )
+  return <BodyWrapper className={className}>{children}</BodyWrapper>
```

Comment:
`Version` should never have been here - now that we have a `Footer` component it makes much more sense to home it there